src/server/game/Entities/Unit/Unit.h

```diff
@@ -1747,6 +1753,8 @@ class TC_GAME_API Unit : public WorldObject
         virtual bool CanEnterWater() const = 0;
         virtual bool CanSwim() const;
 
+        float GetAdvFlyingVelocity() const;
```

Comment:
GetAdvFlightVelocity?

Reply:
No, Blizz call it "AdvFly", short for "Advanced Flying" (check opcodes names)

Reply:
oh so it's a shortening, in that case we gucci